### PR TITLE
pktgen_utils:Added support for 64k pagesize

### PR DESCRIPTION
**Pull request by zhenyzha**
For the arm platform, there are two pagesize package, 
Update the pagesize package acquisition method by the current 
guest base page size.

ID:2961
Signed-off-by: zhenyzha <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
@yanglei-rh @PaulYuuu Could you pls help review when you are free. thanks.
` (1/1) Host_RHEL.m10.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.aarch64.page_64k.io-github-autotest-qemu.pktgen_burst_test.arm64-pci: PASS (149.86 s)`

**Comment by yanglei-rh** [comment]
Hi @zhenyzha I think @quanwenli might be more suitable to help review this patch.

**Review by PaulYuuu — APPROVED** [review]

File: generic/tests/pktgen_perf.py [hunk 1]
```diff
@@ -67,12 +67,18 @@ def init_vm_and_login(test, params, env, result_file, pktgen_runner):
         process.system_output("numactl --hardware")
         process.system_output("numactl --show")
         _pin_vm_threads(params.get("numa_node"))
-        guest_ver = session_serial.cmd_output(guest_ver_cmd)
+        guest_ver = session_serial.cmd_output(guest_ver_cmd).strip()
         result_file.write("### guest-kernel-ver :%s" % guest_ver)
```
Comment (quanwenli) [inline generic/tests/pktgen_perf.py:71]: This resulted in a single-line output
"### guest-kernel-ver :6.11.0-26.el10.x86_64Script:pktgen_perf Category:tx"

Comment (zhenyzha) [reply generic/tests/pktgen_perf.py:71]: Already corrected

**Comment by zhenyzha** [comment]
@quanwenli Could you pls help review again when you are free. thanks.
` (1/1) Host_RHEL.m10.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.aarch64.page_64k.io-github-autotest-qemu.pktgen_burst_test.arm64-pci: PASS (154.90 s)`

**Comment by quanwenli** [comment]
It passed, and the results look good to me. LGTM 

 (1/1) 82599.bridge_test.1q.acceptance.repeat1.Host_RHEL.m10.u0.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: STARTED
 (1/1) 82599.bridge_test.1q.acceptance.repeat1.Host_RHEL.m10.u0.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (367.36 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Review by quanwenli — APPROVED** [review]

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**